Routes/CommerceRoutes.cs

```diff
@@ -70,6 +70,20 @@ public class CommerceRoutes : IRouteProvider {
                             {"area", "Nwazet.Commerce"}
                         },
                         new MvcRouteHandler())
+                },
+                new RouteDescriptor {

```

Comment:
THe default route does not seem to work. I am not really sure why, so I'll look into it, but I will keep the custom route as defined here for now.